src/Files.App/Views/LayoutModes/StandardLayoutMode.cs

```diff
@@ -85,15 +85,15 @@ protected virtual async void ItemManipulationModel_RefreshItemThumbnail(object?
 
 		protected virtual async Task ReloadSelectedItemIconAsync()
 		{
-			ParentShellPageInstance.FilesystemViewModel.CancelExtendedPropertiesLoading();
+			ParentShellPageInstance?.FilesystemViewModel.CancelExtendedPropertiesLoading();
 			ParentShellPageInstance.SlimContentPage.SelectedItem.ItemPropertiesInitialized = false;
```

Comment:
The process should be aborted if `ParentShellPageInstance` is null, because it makes no sense to consider null only for line 88.

Reply:
👍